pnpm-lock.yaml

```diff
@@ -18068,7 +18068,7 @@ packages:
     engines: {node: ^14.15.0 || ^16.10.0 || >=18.0.0}
     dependencies:
       '@jest/types': 29.6.3
-      '@types/node': 18.19.1
+      '@types/node': 20.10.5
       chalk: 4.1.2
```

Comment:
All of the `@types/node` changes should be reverted.
cc @jatgarg 
Lock files can be annoying to review, but we must.

Reply:
https://github.com/microsoft/FluidFramework/pull/19746